test/NuGet.Core.Tests/NuGet.PackageManagement.Test/NuGet.PackageManagement.Test.csproj

```diff
@@ -11,6 +11,7 @@
   <ItemGroup>
     <ProjectReference Include="$(TestUtilitiesDirectory)Test.Utility\Test.Utility.csproj" />
     <ProjectReference Include="$(NuGetCoreSrcDirectory)NuGet.PackageManagement\NuGet.PackageManagement.csproj" />
+    <ProjectReference Include="..\NuGet.Configuration.Test\NuGet.Configuration.Test.csproj" />
```

Comment:
Why do we need this?

We tend to add common methods in the Test.Utility assembly.

Reply:
I used `SettingsTestUtils.CreateConfigurationFile` method from `NuGet.Configuration.Test`. It already exist there, not new method.